### PR TITLE
1719 support more styles to toggle buttons

### PR DESCRIPTION
**Pull request by Doodledun**

### CODE REVIEW

File: ipywidgets/widgets/widget_selection.py [hunk 1]
```diff
@@ -384,9 +385,22 @@ class ToggleButtonsStyle(DescriptionStyle, CoreWidget):
     button_width: str
         The width of each button. This should be a valid CSS
         width, e.g. '10px' or '5em'.
+
+    button_color: str or None
+        The color of each button. This should be a valid CSS color unit,
+        e.g. 'yellow', '#FFFF00', '255,255,0'.
```
Comment (jasongrout) [inline ipywidgets/widgets/widget_selection.py]: The rgb color should be of the form `rgb(255,255,0)`, right? https://developer.mozilla.org/en-US/docs/Web/CSS/color

Comment (Doodledun) [reply ipywidgets/widgets/widget_selection.py]: Yes, you are totally right. Thanks for the catch! I will make updates accordingly.

**Comment by jasongrout** [comment]
For future reference, this is fixing #1719.

**Comment by jasongrout** [comment]
Testing the examples given reveals that the color trait won't take something like `'rgb(255,255,0)'`, so I deleted that from the examples.

**Comment by jasongrout** [comment]
Thanks again!

**Comment by ankostis** [comment]
Is this supposed to be merged in `ipywidgets-7.4.2` (``jupyterlab-v0.35.4`)?
```python
>>> display(widgets.Button().style.keys)
'_model_module',
 '_model_module_version',
 '_model_name',
 '_view_count',
 '_view_module',
 '_view_module_version',
 '_view_name',
 'button_color',
 'font_weight']
>>> display(widgets.ToggleButton().style.keys)
['_model_module',
 '_model_module_version',
 '_model_name',
 '_view_count',
 '_view_module',
 '_view_module_version',
 '_view_name',
 'description_width']
```

**Comment by jasongrout** [comment]
> Is this supposed to be merged in `ipywidgets-7.4.2` (`jupyterlab-v0.35.4`)?

This is answered in https://github.com/jupyter-widgets/ipywidgets/issues/3268#issuecomment-913758351

**Pull request merged**